modules/dreamview/frontend/src/renderer/ground.js

```diff
@@ -23,20 +23,71 @@ export default class Ground {
             return false;
         }
 
-        const { xres, yres, mpp } = PARAMETERS.ground;
+        if (!this.render(coordinates)) {
+            return false;
+        }
+
+        this.initialized = true;
+        return true;
+    }
+
+    update(world, coordinates, scene) {
+    }
+
+    updateImage(coordinates, mapName) {
+        if (!this.mesh) {
+            return;
+        }
+        const dir = this.titleCaseToSnakeCase(mapName);
+        /*import(`assets/map_data/${dir}/background.jpg`)
+        .then(ground => {
+            console.log("updating ground image with " + dir);
+            loadTexture(ground, texture => {
+                this.mesh.material.map = texture;
+                this.render(coordinates, dir);
+            });
+        })*/
+
+        const server = 'http://' + window.location.hostname + ':8888';
+        const imgUrl = server + '/assets/map_data/' + dir + '/background.jpg';
```

Comment:
I think the server part can be omitted?

Reply:
Server port needs to be fixed as the dist always serves from 8888 no matter dev or prod mode